apps/backend/src/mcp/resources/proxy_configs.py

```
@@ -0,0 +1,634 @@
+        directory_resource = await _get_directory_listing_resource(
+            device, '/mnt/appdata/swag/nginx/proxy-confs'
+        )
+        
+        if 'files' not in directory_resource:
+            return directory_resource  # Return error
+        
+        files = directory_resource['files']
+        
+        # Calculate summary statistics
+        total_configs = len(files)
+        services = set(f['service_name'] for f in files)
```

Ruff (0.12.2)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)